src/views/HomePage.vue

```diff
@@ -461,6 +475,14 @@ async function openToast(): Promise<void> {
 async function onAddressSelected(addr: Address): Promise<void> {
   addressInput.value.setValue(`${addr.address} ${addr.address2 ? addr.address2 : ''}, ${addr.postalCode} ${addr.city}, ${addr.country}`);
 }
+
+async function validationFunction(code: Array<string>): Promise<boolean> {
+  if (code.length === VALID_CODE.length && code.every((value, index) => value === VALID_CODE[index])) {
```

Comment:
You can just
`return code.length === VALID_CODE.length && code.every((value, index) => value === VALID_CODE[index]);`